.github/workflows/swift-toolchain.yml

```diff
@@ -2118,9 +2118,117 @@ jobs:
           symbolsFolder: ${{ github.workspace }}/BinaryCache
           searchPattern: '**/*.exe'
 
+  debugging_tools:
+    name: Debugging Tools
+    needs: [context, compilers, devtools, sdk]
+    runs-on: ${{ needs.context.outputs.windows_build_runner }}
+    env:
+      TOOLCHAIN_PATH: ${{ github.workspace }}/BuildRoot/Library/Developer/Toolchains/unknown-Asserts-development.xctoolchain/usr/bin
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - arch: amd64
+            cpu: x86_64
+            triple: x86_64-unknown-windows-msvc
+
+          # TODO(thebrowsercompany/swift-build#125): Fix the arm64 build and uncomment this.
+          # - arch: arm64
+          #   cpu: aarch64
+          #   triple: aarch64-unknown-windows-msvc
+
+    steps:
+      - name: Download compilers
+        uses: actions/download-artifact@v4
+        with:
+          name: compilers-${{ matrix.arch }}
+          path: ${{ github.workspace }}/BuildRoot/Library
+
+      - name: Download devtools
+        uses: actions/download-artifact@v4
+        with:
+          name: devtools-${{ matrix.arch }}
```

Comment:
Do we not need the amd64 devtools always as we are cross-compiling x64 -> arm64 and x64 -> x64?

Reply:
Same here, mind if I address this while investigating issue # 125?

Reply:
Likewise.

Reply:
Done